tests/ExecutorTest.php

```diff
@@ -152,12 +152,12 @@ public function testBuild(): array
         $response = $this->client->call(Client::METHOD_GET, '/runtimes', [], []);
         $this->assertEquals(200, $response['headers']['status-code']);
         $this->assertEquals(1, count($response['body']));
-        $this->assertEquals('test-build', $response['body'][0]['name']);
+        $this->assertStringContainsString('test-build', $response['body'][0]['name']);
 
         /** Get runtime */
         $response = $this->client->call(Client::METHOD_GET, '/runtimes/test-build', [], []);
         $this->assertEquals(200, $response['headers']['status-code']);
-        $this->assertEquals('test-build', $response['body']['name']);
+        $this->assertStringContainsString('test-build', $response['body']['name']);
```

Comment:
If there is stringEndsWith, lets use that. Will make it clear there is prefix we ignore. If not, all good.